cves/kernel/CVE-2021-23133.yml

```diff
@@ -395,8 +419,10 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+    Attacker is able to gain root privileges from an external process, the
+    farthest depth you can go.
```

Comment:
You should talk about how defense in depth would have prevented this. What additional layer of defense should have been in place to help mitigate this issue?